packages/components/tag/src/Tag.styled.tsx

```diff
@@ -0,0 +1,70 @@
+import tokens from '@contentful/f36-tokens';
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+import type { EntityStatus, TagType } from './types';
+
+export interface TagProps {
+  tagType?: TagType;
```

Comment:
@mshaaban0 made a proposal of the properties naming convention. We might go with `variant` over here? and the `entityStatusVariant` I guess?

Reply:
Yeah, I think this PR is just to validate the package separation, we might want to address broader changes in other PRs

Reply:
I'd agree on the naming though before merging :)

Reply:
I agree that we might want to change prop names here, but I'd rather merge this and follow-up later will all other changes.

Regarding, `Tag | Pill | Badge`:

For me, it makes sense to rename `Tag` to `Badge` but I would avoid renaming `Pill` to `Tag` and confusing people with a swap for names.

Reply:
Left a comment in the proposal: https://github.com/contentful/forma-36/issues/841#issuecomment-791404659

Reply:
Updated according to the proposal